docs/docs/spec.md

```diff
@@ -244,7 +244,10 @@ exceptions in the subsequent sections:
 
 -   an array component type
 
--   an array creation expression
+    > Note that this rule recognizes the annotations in `@Nullable String []`
```

Comment:
```suggestion
    > Note that this rule recognizes the annotations in `@Nullable String[]`
```

Should that space be removed, to be consistent with the other `String[` occurrences?
Following what GJF does sounds okay to me, but this now looks inconsistent.